src/schema/sources.ts

```diff
@@ -604,6 +633,8 @@ export enum SourcePermissions {
   Edit = 'edit',
 }
 
+const alwaysExcludeSources = ['unknown', 'community'];
```

Comment:
Always excluded for what? Maybe just naming a bit weird, we can also move it to `src/types` as global const.

Reply:
I had them per query at first, you think it's better?
As it's only for 2 queries now.

Reply:
I am not sure what it means, for `unknown` is fine because it is a "ghost" source but for example `community` we don't exclude on different places.

Reply:
Someone could take it and use it as "this sources need to be excluded all the time".

Reply:
Yeah let's just move it to both functions then it's more explicit :)

Reply:
Sounds good, just add it before merging.

Reply:
Forgot to push, added now :)